source/routing/tests/routing_api_test.cpp

```diff
@@ -3139,22 +3146,22 @@ BOOST_FIXTURE_TEST_CASE(night_bus_should_treated_as_way_later_anti_clockwise, Ni
 
 BOOST_FIXTURE_TEST_CASE(night_bus_filter_should_be_order_agnostic, Night_bus_fixture) {
     {
-        nr::RAPTOR::Journeys journeys = {j1, j2};
+        nr::RAPTOR::Journeys journeys = {j0, j1, j2};
         nr::filter_late_journeys(journeys, filter_params);
```

Comment:
OK, `filter_late_journeys` calls `get_best_journeys`

Reply:
Ah yes, sorry !
It was straightforward to me as I dived into the code. I will just copy your explanation on top-description to help next reviewers :+1: